packages/web-app-files/src/components/SideBar/Shares/FileShares.vue

```diff
@@ -176,6 +216,12 @@ export default {
         role
       }
     },
+    collaborators_avatar() {
+      const result = []
+      console.log(this.collaborators)
+      this.collaborators.forEach(c => result.push({ ...c.collaborator, shareType: c.shareType }))
```

Comment:
Instead of building a result in a const, you can just
```
return this.collaborators.map(c => {
  ...c.collaborator,
  shareType:  c.shareType
})
```

Reply:
nope the '...' doesnt work on map

Reply:
Why? The following works fine:
```js
collaborators_avatar() {
      return this.collaborators.map(c => {
        return {
          ...c.collaborator,
          shareType: c.shareType
        }
      })
    },
```

Why is the computed prop name in snake case by the way? 🤔